ethereum-bridged-carbon/src/MossBatch.ts

```diff
@@ -0,0 +1,30 @@
+import {CreateBatchCall } from '../generated/MossBatch/MossBatch'
+import { toDecimal } from '../../lib/utils/Decimals'
+import { loadOrCreateTransaction } from './utils/Transactions'
+import {Batch} from "../generated/schema";
+
+
+export function handleBatchCall(call: CreateBatchCall): void {
+  
+  let transaction = loadOrCreateTransaction(call.transaction, call.block)
+
+  let id = transaction.timestamp.toString() + call.inputs._serialNumber
+
+  let batch = Batch.load(id)
+  if (batch == null) {
```

Comment:
The only thing i am curious here is - I've seen mostly within subgraph repositories (ours and OHM's) that we are usually creating an Entity with empty values in case of null and then override those values regardless whether there is an existing entity or not.

This will work if we start syncing all over again, I am just not sure if there is any scenario where we will be overriding already existing Entities - and if there is such scenario, this won't work since the override will be ignored if there is an existing Entity.

Other than that, it looks good to me 💪